app/views/categories/index.html.erb

```diff
@@ -0,0 +1,26 @@
+
+
+<div class="c-container">
+ <div class="header">
+  <%= link_to "New", new_user_category_path, class: "btn" %>
+  <h3 class="title">CATEGORY</h3>
+  <div class="sign-out">
+  <%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "btn" %>
+  </div>
+  </div>
+
+<div id="categories">
+ 
+  <% if @categories.empty? %>
+      <p>No categories found.</p>
+    <% else %>
+      <% @categories.each do |category| %>
+        <%= render category %>
+      <% end %>
+    <% end %>
+</div>
+<div class="new">
+<%= link_to "New category", new_user_category_path, class: "new-data" %>
```

Comment:
- [ ] From the design perspective, I would suggest to you add a bit of improvement on this page. I would suggest that you align the `Add category` button at the bottom of the page in the mobile first app view as we find on the design given. Please find the comparison below 👇🏾 

**Your design**
<img width="378" alt="Screen Shot 2023-09-16 at 17 29 02" src="https://github.com/isume295/Budget-app/assets/90222110/696405a1-df05-4966-b7fd-f121b2bb46de">

**Given design**
<img width="268" alt="Screen Shot 2023-09-16 at 17 29 12" src="https://github.com/isume295/Budget-app/assets/90222110/7d6cb3ef-eaee-4a85-851d-8400b2031972">

Reply:
Thank you